lib/fetchers/TokenFetcher.ts

```diff
@@ -58,22 +58,12 @@ export class TokenFetcher {
     }
   };
 
-  public resolveToken = async (chainId: ChainId, address: string): Promise<Currency> => {
+  public getTokenByAddress = async (chainId: ChainId, address: string): Promise<Currency | undefined> => {
```

Comment:
nit: little hesitant about changing this to return undefined but i think we only use it for the emitted metrics so shouldnt rly be a problem